raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientTest.java

```diff
@@ -92,25 +97,25 @@ public void testNodeDirectoryId() {
     }
 
     @ParameterizedTest
-    @ValueSource(booleans = { true, false })
-    public void testInitializeSingleMemberQuorum(boolean withKip853Rpc) throws IOException {
+    @EnumSource(value = RaftProtocol.class, names = {"KIP_595_PROTOCOL", "KIP_853_PROTOCOL"})
```

Comment:
As @ahuang98 mentioned. Let's try to be consistent between `KafkaRaftClientTest` and `KafkaRaftClientSnapshotTest`. Ideally we would test all `RaftProtocol` but I understand if you want to consider that outside the scope of this PR if it is too difficult to change and make all of these tests pass.

Reply:
I think it could be done in a follow-up PR.
If you agree, I will open a ticket to trace it.

Reply:
Please ignore the above comment.
I tried to enable all raft protocols for `KafkaRaftClientTest`, and all tests passed.
I will include this change in the next commit.